src/graph/Ros2PublishPointsNode.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright 2022 Robotec.AI
```

Comment:
I wanted to follow `NodesCore` style. I think it should be consistent. In the near future, we are going to separate PCL nodes and another `NodesX.hpp` will be created. There are 2 options:
- One `NodesX.hpp` file that accumulates all nodes definitions corresponding to the public header, and separate `YNode.cpp` implementations for these nodes (current solution).
- Separate header files for each node and separate implementations.

What do you think?

Reply:
You're right, my proposition is inconsistent in terms of where to look for the implementation of a given node (core in separate files, extensions in their collective file). IMHO it's better to keep your approach.

> Separate header files for each node and separate implementations.

This makes editing all nodes much more annoying, so I'd like to avoid that for the time we still crystalize the graph/nodes system.